lib/msf/core/module/options.rb

```diff
@@ -63,7 +61,6 @@ def register_evasion_options(options, owner = self.class)
   #
   def register_options(options, owner = self.class)
     self.options.add_options(options, owner)
-    self.datastore.import_options(self.options, 'self', true)
     import_defaults(false)
```

Comment:
I statically verified this line deletion, it seemed like `import_defaults(false)` already calls this line under the hood and it seemed good to remove

https://github.com/rapid7/metasploit-framework/blob/d88d4d93296c34f3f336f4d4a7f29a963de225e9/lib/msf/core/module/data_store.rb#L16-L20